src/routes/accounts.js

```diff
@@ -0,0 +1,61 @@
+const express = require('express');
+const uuidv1 = require('uuid/v1');
+const router = express.Router();
+
+const {users, writeData } = require('../utility/dataAdapter');
+
+router.post('/login', (req, res) => {
+    const username = req.body.email;
+    const password = req.body.password;
+
+    if(users.find((u) => u.email === username && u.password === password)){
+        res.json({
+            success: true,
+            message: 'logged in successfully',
+            session_id: uuidv1()
+        });
+    } else {
+        res.json({
```

Comment:
Why not send error code maybe 400 or 401 or 500 as response than 200 with success message